### DISCUSSION TITLE
Deploy Winmerge with all settings?

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1959] [General]
On my PC I configured a lot of stuff for Winmerge - settings, plugins, filters, ...

Now some colleagues want to use the same configuration. So what I have to do to "deploy" my "entire package"? Only copy the program folder? or the settings and the Appdata and the registry and the ...??

Thanks for hints!

**Comment by sdottaka** [comment]
Please export the configuration information either by exporting the following registry key or using the Options window to export to an INI file:

Registry key: `HKEY_CURRENT_USER\Software\Thingamahoochie\WinMerge`

Since the exported file may contain the username, you may need to replace the username.

other files are as follows:

Plugins: `C:\Program Files\WinMerge\MergePlugins`
File filters: `C:\Program Files\WinMerge\Filters and Documents\WinMerge\Filters`
Color schemes: `C:\Program Files\WinMerge\ColorSchemes`
Downloaded plugin-related files (e.g. Apache Tika): `%APPDATA%\WinMerge\Commands`

**Comment by Pierre-de-la-Verre** [comment]
Thanks!
